README.md

```diff
@@ -26,6 +26,11 @@ Contributions<sup>[2]</sup> and corrections are welcome. Please talk to us
 about your suggestions at [dev@geode.apache.org](mailto:dev@geode.apache.org)
 or submit a [pull request](https://github.com/apache/geode/pull/new/develop).
 
+# Apache Geode Version
```

Comment:
I think we should just tell people to check out the master branch, which should be the default. That will have the examples that run against the latest geode release. 

We shouldn't point people at the snapshot releases per apache policy.

Reply:
fixed readme